src/wallet/rpcwallet.cpp

```diff
@@ -4527,36 +4527,36 @@ static RPCHelpMan upgradewallet()
 #ifdef ENABLE_EXTERNAL_SIGNER
 static RPCHelpMan walletdisplayaddress()
 {
-    return RPCHelpMan{
-        "walletdisplayaddress",
-        "Display address on an external signer for verification.\n",
+    return RPCHelpMan{"walletdisplayaddress",
+        "\nDisplay address on an external signer for verification.\n",
```

Comment:
nit: I think the leading and trailing newlines should be the responsibility of RPCHelpMan. Also, they wouldn't make sense if the help is exported as json.